torch/_export/non_strict_utils.py

```diff
@@ -315,6 +312,10 @@ def make_constraints(
     range_constraints = {
         symbol: inline_constraints[symbol] for symbol in inline_constraints
     }
+    if dynamic_shapes is True:
+        dynamic_shapes = _tree_map_with_path(
+            lambda path, t, shape: None, combined_args, combined_args
```

Comment:
So `None` for an entire tensor, right? Not `None` for individual dims. This will somehow cause everything to be marked auto-dynamic? Can you link to the code that does that?

Reply:
Add a comment too, please.

Reply:
Oh, the code that does that is just process_dynamic_shapes doing nothing for Nones, with assume_static_by_default=False

Reply:
Sorry I think I asked this before, but I forgot what the answer was. What happens when you specify True and also specify Dim constraints for the shape? Do we error or do we intersect them?

Reply:
Ahh we error, nvm